src/main/java/mate/academy/hw01/deepClone/Main.java

```diff
@@ -0,0 +1,99 @@
+package mate.academy.hw01.deepClone;
+
+import java.util.Objects;
+
+public class Main {
+    public static void main(String[] args) throws Exception {
+        Animals myAnimals = new Animals();
+        myAnimals.name = "Cat";
+        Dog newObject = new Dog();
+        newObject.age = 3;
+        myAnimals.newObject = newObject;
+
+        Animals myObject2 = myAnimals.clone();
+        System.out.println("myObject2.name = " + myObject2.name);
+        System.out.println("myObject2.age = " + myObject2.newObject.age);
+        System.out.println(myAnimals.hashCode() == myObject2.hashCode());
+
+//        myObject2.name = "Tiger";
```

Comment:
Remove comments

Reply:
I did it: https://github.com/MateAcademy/lessons/commit/31c5735ff1a9b9580e0ee568142e391ec63a8a61